### DISCUSSION TITLE
New endpoint GET events/{id}/layers ?

### DISCUSSION THREAD

**Discussion by jmargutt** [rodekruis/ADA-API#19] [Ideas]
@gulfaraz would it be lots of work to add an endpoint to list the existing event layers?
As a use case, I'm thinking about
1. getting statistics on what is currently on the ADA portal
2. having a more elegant way to check what each event has (instead of just trying each layer and getting "layer not found")

**Comment by gulfaraz** [comment]
What would the response look like? An array of available layer names as a part of the event object? Or a json with layer names as keys and boolean values? An example would be helpful.

**Reply by jmargutt** [reply]
I would expect a list of jsons with layer name, info and create/update time
```
[
  {
    "createdAt": "2022-08-23T13:16:44.491Z",
    "updatedAt": "2022-08-23T13:16:44.491Z",
    "information": "this is the admin-1 layer",
    "name": "admin-1"
  },
  {
    "createdAt": "2022-08-23T13:16:44.491Z",
    "updatedAt": "2022-08-23T13:16:44.491Z",
    "information": "this is the admin-2 layer",
    "name": "admin-2"
  }
  ]
```

**Reply by gulfaraz** [reply]
It's not trivial but it's simple enough. I've converted this discussion to [a backlog item](https://github.com/rodekruis/ADA-API/issues/21).
